### PR TITLE
Fix #1147 README update

### LINKED ISSUE #1147: Mention the version of python used in the project in the README file.
Mention the version of python used in the project in the README file.

### PR DESCRIPTION
**Pull request by HargovindArora**
Add python 2.x requirement in README.md Fix #1147.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ EvalAI is an open source web application that helps researchers, students and da
 Setting up EvalAI on your local machine is really easy.
 Follow this guide to setup your development machine.
 
-1. Install [git], [postgresql] version >= 9.4, [RabbitMQ] and [virtualenv], in your computer, if you don't have it already.
+1. Install [python]2.x for this project as we don't support Python3, [git], [postgresql] version >= 9.4, [RabbitMQ] and [virtualenv], in your computer, if you don't have it already.
```
Comment (deshraj) [inline README.md]: Change `[python]2.x for this project as we don't support Python3` to `[python 2.x]`

File: README.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ Follow this guide to setup your development machine.
 
 If you are interested in contributing to EvalAI, follow our [contribution guidelines](https://github.com/Cloud-CV/EvalAI/blob/master/.github/CONTRIBUTING.md).
 
+[python]: https://www.python.org/downloads/
```
Comment (deshraj) [inline README.md]: Please change the link to download Python 2.7.

File: README.md (hunk 1 above)
Comment (HargovindArora) [inline README.md]: @deshraj OK

**Comment by RishabhJain2018** [comment]
@HargovindArora Now, you have to `rebase` your branch from master and also squash all the commits into a single commit with a meaningful commit message like `Update Readme: Adds python version with download link.` and then force push again on this branch.

**Comment by HargovindArora** [comment]
@RishabhJain2018 Anymore changes required?

File: apps/challenges/serializers.py [hunk 3]
```diff
@@ -42,15 +42,12 @@ def __init__(self, *args, **kwargs):
             challenge = context.get('challenge')
```
Comment (RishabhJain2018) [inline apps/challenges/serializers.py]: This file should not be a part of this PR. It is already fixed.

File: tests/unit/challenges/test_views.py [hunk 4]
```diff
@@ -1044,7 +1044,6 @@ def test_get_challenge_phase(self):
                 "codename": "Phase Code Name",
```
Comment (RishabhJain2018) [inline tests/unit/challenges/test_views.py]: Same as above.

**Comment by HargovindArora** [comment]
@RishabhJain2018 Sure.

File: README.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ EvalAI is an open source web application that helps researchers, students and da
 Setting up EvalAI on your local machine is really easy.
 Follow this guide to setup your development machine.
 
-1. Install [git], [postgresql] version >= 9.4, [RabbitMQ] and [virtualenv], in your computer, if you don't have it already.
+1. Install [python]2.x, [git], [postgresql] version >= 9.4, [RabbitMQ] and [virtualenv], in your computer, if you don't have it already.
```
Comment (the-dagger) [inline README.md]: @HargovindArora add a space after [python] and 2.x.

Comment (aayusharora) [inline README.md]: @dagger, this seems correct according to English rules.
`There should be a space on the right of comma but no space on the left of comma.`

Comment (HargovindArora) [inline README.md]: @the-dagger okay.

**Pull request merged**